src/store/lists/mutations.js

```diff
@@ -32,6 +32,26 @@ export default {
 
     state.lists.splice(index, 1, list);
   },
+
+  addTagToListLocally(state, { listId, tagTitle }) {
+    const index = state.lists.findIndex(localList => localList.id === listId);
+    const { tags } = state.lists[index];
```

Comment:
why not just find and after taking .tags?

Reply:
i guess because i'm not strong enough in array methods :))

Reply:
it was hard to understand